docs/lib/predictions/fragments/ios/getting-started/20_preReq.md

```diff
@@ -0,0 +1,2 @@
+* An iOS application targeting at least iOS 13.0 with Amplify libraries integrated
```

Comment:
Should we tell people what happens on iOS <13? Say, what happens?

Reply:
The cocoapod spec basically says iOS 13, if they try to run it, they won't even be able to include the pod at the moment... i think it's fine to leave as is.